setup.py

```diff
@@ -29,7 +29,7 @@ def build_media_pattern(base_folder, file_extension):
 
 setup(
     name = "django-wiki",
-    version = "0.0.2",
+    version = "0.0.3",
```

Comment:
This version bumping makes me nervous: the upstream repo is at "0.1.dev"  I guess we've pretty much abandoned any hope of re-joining master some day.